assets/scss/form.scss

```diff
@@ -2,7 +2,7 @@
   background-color: $offwhite;
   border: 0;
   font-weight: normal;
-  height: 2.5rem;
+  height: 2.625rem;
```

Comment:
Just a detail, but I think this makes the search box 2 pixels too tall, compared to the design.

Reply:
@mirjamverloop I reverted to this style since reducing the height/padding of the nav was causing the tiny gap between the active link and the bottom border to show again

Reply:
ok I've reduced the height and added margin to keep the blue underlines in the right position